src/hardware/pcspeaker_impulse.h

```diff
@@ -52,7 +52,9 @@ class PcSpeakerImpulse final : public PcSpeaker {
 	static constexpr char device_name[] = "PCSPEAKER";
 	static constexpr char model_name[]  = "impulse";
 
-	static constexpr int16_t positive_amplitude = 20000;
+	// Amplitude constants
+	static constexpr float pwm_scalar = 0.5f; // accomodate a full doubling
```

Comment:
So `pwm_scalar` was change to 0.72 above, why not here as well?

Reply:
The impulse PC Speaker code produces louder samples - but the adjustment was also made to match the hardware levels (comment also added).